ice_model.F90

```diff
@@ -609,13 +608,13 @@ subroutine ice_model_init( Ice, Time_Init, Time, Time_step_fast, Time_step_slow,
    endif
 
 else
-    if (file_exist('INPUT/ice_model.res')) then
+    if (file_exists('INPUT/ice_model.res')) then
```

Comment:
This is the "native formatted restart files", which are binaries and are no longer supported (and probably no longer exist)